src/core/lib/iomgr/polling_entity.cc

```diff
@@ -61,8 +61,16 @@ bool grpc_polling_entity_is_empty(const grpc_polling_entity* pollent) {
 void grpc_polling_entity_add_to_pollset_set(grpc_polling_entity* pollent,
                                             grpc_pollset_set* pss_dst) {
   if (pollent->tag == GRPC_POLLS_POLLSET) {
+#ifdef GRPC_CFSTREAM
```

Comment:
yeah, I do agree that having too many #ifdef in the code is not a good  idea.
having `using_cfstream` flag sounds like a good idea.

Reply:
I would just leave the cfstream case. Having a global flag does not quite work here since I do see people running CFStream client and POSIX server in the same binary.